### DISCUSSION TITLE
Archive Dashboards

### DISCUSSION THREAD

**Discussion by Floxinium** [grafana/grafana#39180] [Dashboards, Panels & Visualizations]
Hi,

additionally to deleting dashboards I would love to see an option for archiving dashboards.

reason:
At my company we have tons of dashboards and after months/years go by you'll start to collect a fair amount of unmaintained dashboards. But deleting them can often be scary because you don't know if that dashboard might be needed in some way again or if maybe just a small update is required to bring it back to life. Also asking your colleague can also be inconvenient if you need to ask 10 different colleagues about their dashboards and then maybe take a long time to respond. Or the colleague left the company already.
Now straight up deleting unmaintained dashboards can also come with it's own set of issues if it turns out that the dashboard was in fact still needed or useful in some way.

How I would imagine the archiving function works in detail:
A button next to the "delete dashboard" button in the dashboard settings:
![image](https://user-images.githubusercontent.com/8218576/133260707-95b87028-dd0f-4d1d-ae07-503e6bc3d36a.png)
When you click it you'll see a confirm dialog just like with the delete button.
Once the dashboard is archived it won't show up in the dashboard search anymore but you can view archived dashboards by clicking a checkbox "show archived dashboards" here:
![image](https://user-images.githubusercontent.com/8218576/133261141-4d492369-fcba-49ad-9476-c40911d74604.png)
The archived dashboards stay in the same folder as they were archived in.
Archived dashboards can be unarchived again or you can fully delete them.
You can view archived dashboard just like normal dashboards.
When viewing an archived dashboard maybe put an info somewhere that this is indeed an archived dashboard.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
